cosmo_tester/test_suites/snapshots/multi_tenant_snapshot_test.py

```diff
@@ -123,7 +124,7 @@ def test_restore_snapshot_and_agents_upgrade_multitenant(
 
 @pytest.fixture(
         scope='module',
-        params=['master', '4.1', '4.0.1'])
+        params=get_multi_tenant_versions_list())
```

Comment:
Agreed, and will probably be useful in the multi-tenant conversion stuff that I'll be stacking on after this.